api/api.go

```diff
@@ -215,103 +215,113 @@ func NewServerRoutes(s Server) *mux.Router {
+	m.Handle(lib.AEProfilePoster.String(), s.Middleware(proh.PosterHandler))
+
+	// remote endpoints
 	m.Handle(lib.AEFeeds.String(), s.Middleware(remClientH.FeedsHandler))
 	routeParams = newrefRouteParams(lib.AEPreview, false, false, http.MethodGet, http.MethodPost)
 	handleRefRoute(m, routeParams, s.Middleware(remClientH.DatasetPreviewHandler))
+	m.Handle(lib.AERegistryNew.String(), s.Middleware(rch.CreateProfileHandler))
+	m.Handle(lib.AERegistryProve.String(), s.Middleware(rch.ProveProfileKeyHandler))
+	m.Handle(lib.AESearch.String(), s.Middleware(sh.SearchHandler))
 
+	// working directory endpoints
 	routeParams = newrefRouteParams(lib.AEStatus, false, false, http.MethodGet, http.MethodPost)
 	handleRefRoute(m, routeParams, s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.status")))
-	routeParams = newrefRouteParams(lib.AEWhatChanged, false, false, http.MethodGet, http.MethodPost)
-	handleRefRoute(m, routeParams, s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.whatchanged")))
+	m.Handle(lib.AECanInitDatasetWorkDir.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.caninitdatasetworkdir")))
 	routeParams = newrefRouteParams(lib.AEInit, true, false, http.MethodPost)
 	handleRefRoute(m, routeParams, s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.init")))
-	m.Handle(lib.AECanInitDatasetWorkDir.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.caninitdatasetworkdir")))
 	m.Handle(lib.AECheckout.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.checkout"))).Methods(http.MethodPost)
 	m.Handle(lib.AERestore.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.restore"))).Methods(http.MethodPost)
 	m.Handle(lib.AEFSIWrite.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.write"))).Methods(http.MethodPost)
 	m.Handle(lib.AEFSICreateLink.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.createlink"))).Methods(http.MethodPost)
 	m.Handle(lib.AEFSIUnlink.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "fsi.unlink"))).Methods(http.MethodPost)
 
-	renderh := NewRenderHandlers(s.Instance)
-	routeParams = newrefRouteParams(lib.AERender, false, false, http.MethodGet, http.MethodPost)
-	handleRefRoute(m, routeParams, s.Middleware(renderh.RenderHandler))
+	// sync/protocol endpoints
+	if cfg.Remote != nil && cfg.Remote.Enabled {
+		log.Info("running in `remote` mode")
+
+		remh := NewRemoteHandlers(s.Instance)
+		m.Handle(lib.AERemoteDSync.String(), s.Middleware(remh.DsyncHandler))
+		m.Handle(lib.AERemoteLogSync.String(), s.Middleware(remh.LogsyncHandler))
+		m.Handle(lib.AERemoteRefs.String(), s.Middleware(remh.RefsHandler))
+	}
 
-	m.Handle(lib.AEHistory.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "log.history"))).Methods(http.MethodPost)
+	// TODO(aqru): clear up these endpoints up to spec
+
+	m.Handle(lib.AELog.String(), s.Middleware(lib.NewHTTPRequestHandler(s.Instance, "log.history"))).Methods(http.MethodPost)
```

Comment:
We've had a discussion around these, but I don't have all the details to spell them all out on how we will merge these/how they should behave/which we will just drop from the HTTP API.

Reply:
According to https://github.com/qri-io/qri/pull/1730, `rawlogbook` and `logbookssummary` should be dropped from the API, because they're "hard-mode" only. Not totally sure about the other 2.

Reply:
I believe `history` will be the `/log` endpoint with `filter=history` (or `selector=history`, whatever language we choose)